src/scripts/main.js

```diff
@@ -1 +1,88 @@
 'use strict';
+
+let promiseNumber;
+let firstCheck = false;
+let secondCheck = false;
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    promiseNumber = 1;
+    resolve();
+  });
+
+  setTimeout(() => {
+    reject();
+  }, 3000);
+});
```

Comment:
The `promise1` promise will be rejected if no click event occurs within 3 seconds. Ensure that this behavior is intended, as it might lead to unexpected rejection if the user doesn't click in time.